google-cloud-speech/lib/google/cloud/speech/result.rb

```diff
@@ -114,10 +123,40 @@ class Alternative
           ##
           # @private Creates a new Result::Alternative instance.
           def initialize transcript, confidence
-            @transcript  = transcript
+            @transcript = transcript
             @confidence = confidence
           end
         end
+
+        ##
+        # Word-specific information for recognized words. Word information is
+        # only included in the response when certain request parameters are
```

Comment:
"certain request parameters..." Is there any way to provide definite information here?